types/src/proof/definition.rs

```diff
@@ -305,6 +305,66 @@ impl AccumulatorConsistencyProof {
     }
 }
 
+impl FromProto for AccumulatorConsistencyProof {
+    type ProtoType = crate::proto::proof::AccumulatorConsistencyProof;
+
+    fn from_proto(mut proto_proof: Self::ProtoType) -> Result<Self> {
+        let frozen_subtree_roots = proto_proof
+            .take_frozen_subtree_roots()
+            .into_iter()
+            .map(|hash_bytes| HashValue::from_slice(&hash_bytes))
+            .collect::<Result<Vec<_>>>()?;
+
+        let num_siblings = proto_proof.get_num_siblings() as usize;
+        ensure!(
+            num_siblings <= 63,
+            "Too many ({}) siblings in the proof.",
+            num_siblings,
+        );
+        ensure!(
+            proto_proof.get_non_default_siblings().len() <= num_siblings,
+            "Expect {} siblings in total. Got {} non-default siblings.",
+            num_siblings,
+            proto_proof.get_non_default_siblings().len(),
+        );
+        let mut siblings = proto_proof
+            .take_non_default_siblings()
+            .into_iter()
+            .map(|hash_bytes| HashValue::from_slice(&hash_bytes))
+            .collect::<Result<Vec<_>>>()?;
+        while siblings.len() < num_siblings {
+            siblings.push(*ACCUMULATOR_PLACEHOLDER_HASH);
```

Comment:
Is there a special syntax you can use to append (*ACCUMULATOR_PLACEHOLDER_HASH, num_default_siblings) times?

Reply:
Yeah...